### PR TITLE
feat: skip subclass of

### PR DESCRIPTION
**Pull request by TomPavelec**

### CODE REVIEW

File: src/DI/AutoRegistratorExtension.php [hunk 1]
```diff
@@ -16,14 +16,27 @@ final class AutoRegistratorExtension extends CompilerExtension
 
 	private array $defaults = [
 		'scanDirs' => [],
-		'skipDirs' => [
-			'Exception',
```
Comment (paveljanda) [inline src/DI/AutoRegistratorExtension.php:20]: This is a BC break. Please release a major version or revert this change.

File: composer.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
 		}
 	},
 	"require": {
-		"php": ">=7.1",
+		"php": ">=7.4",
```
Comment (paveljanda) [inline composer.json:20]: Well, this is a BC break. Could you release a mojor version? Or revert this change.

File: src/DI/AutoRegistratorExtension.php (hunk 1 above)
Comment (TomPavelec) [reply src/DI/AutoRegistratorExtension.php:20]: Exceptions are now skipped because they are subclass of Throwable

File: composer.json (hunk 2 above)
Comment (TomPavelec) [reply composer.json:20]: PHP 7.4 features was added in version v2.0.0 so this is just fix.
